test/specs/homePage.js

```diff
@@ -0,0 +1,15 @@
+const crossHomePage = require('./crossHomePage'); //importing Page Objects
+
+describe('CrossfadeYT test cases for navigating to other pages', function(){
```

Comment:
Your describes should not be the same, they should be different and describe the group of tests below